.github/actions/job-preamble/action.yaml

```diff
@@ -1,28 +1,141 @@
 name: Job Preamble
 description: Common steps for most jobs
 inputs:
+  free-disk-space:
+    description: 'Free disk space desired in GB (2025-06 ubuntu-24.04 runner starts with 20GB free and we can delete and reach 40GB)'
+    required: false
+    default: 25
   gcp-account:
     description: 'Account to be used to upload tests data'
     required: true
 runs:
   using: composite
   steps:
-    - name: Recover docker image cache space
+    # auth prepares the gcp environment to authenticate.
+    - name: Auth gcloud
+      uses: 'google-github-actions/auth@v3'
+      with:
+        credentials_json: '${{ inputs.gcp-account }}'
+        export_environment_variables: true
+        create_credentials_file: true
+        cleanup_credentials: true
+
+    - name: Check gcloud CLI
+      id: gcloud-check
+      continue-on-error: true
+      shell: bash
+      run: |
+        which gcloud
+
+    # setup-gcloud authenticates gcloud
+    - name: Install gcloud CLI
+      uses: 'google-github-actions/setup-gcloud@v2'
+      with:
+        skip_install: ${{ steps.gcloud-check.outcome != 'failure' }}
+
+    - name: Shim gsutil as gcloud storage
+      continue-on-error: true
+      shell: bash
+      run: |
+        tee -a ~/.boto <<EOF
+        [GSUtil]
+        use_gcloud_storage=True
+        EOF
```

Comment:
Is this part of the change dealing with gcloud and gsutil needed?
It does not seem related to the disk cleanup.

Reply:
https://github.com/stackrox/stackrox/pull/17027/commits/95d5d2ecee780fd75ff4c454ccecebd9f708e0a8